frontend/src/components/Agreements/AgreementChangesResponseAlert/AgreementChangesResponseAlert.jsx

```diff
@@ -0,0 +1,127 @@
+import PropTypes from "prop-types";
+import VanishingAlert from "../../UI/Alert/VanishingAlert";
+import { renderField } from "../../../helpers/utils";
+
+
+/**
+ * Alert for when there are agreement changes in review.
+ * @component
+ * @param {Object} props - The component props.
+ * @param {Object[]} props.changeRequests - The change requests.
+ * @param {boolean} props.isApproveAlertVisible - Whether the approval alert is visible.
+ * @param {boolean} props.isDeclineAlertVisible - Whether the approval alert is visible
+ * @param {Function} props.setIsApproveAlertVisible - The function to set the alert visibility.
+ * @param {Function} props.setIsDeclineAlertVisible - The function to set the Decline alert visibility.
+ * @returns {JSX.Element} - The rendered component.
+ */
+function AgreementChangesResponseAlert({ changeRequests, isApproveAlertVisible, isDeclineAlertVisible, setIsApproveAlertVisible, setIsDeclineAlertVisible }) {
+
+    const approvedRequests = changeRequests.filter((changeRequest) => {
+        if(changeRequest.change_request.status === "APPROVED") {
+            return changeRequest;
+        }
+    });
+
+    const declinedRequests = changeRequests.filter((changeRequest) => {
+        if(changeRequest.change_request.status === "REJECTED") {
```

Comment:
we may have an ENUM we could use instead of "APPROVED" or "REJECTED"

Reply:
No good place for the enum at the moment. We talked about it and we're fine moving without this for now.